apps/organization/src/organization.service.ts

```diff
@@ -268,7 +268,18 @@ export class OrganizationService {
         delete updateOrgDto.logo;
       }
 
-      const organizationDetails = await this.organizationRepository.updateOrganization(updateOrgDto);
+      let organizationDetails;
+      const checkAgentIsExists = await this.organizationRepository.getAgentInvitationDetails(orgId);
+
+      if (!checkAgentIsExists?.connectionInvitation && !checkAgentIsExists?.agentId) {
```

Comment:
Below these two if conditions to optimize the code and avoid checking same condition repeatedly

Reply:
Resolved